src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js

```diff
@@ -0,0 +1,19 @@
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
+import ChallengeOktaVerifyTotpView from './ChallengeOktaVerifyTotpView';
+import ChallengeOktaVerifyFastPassView from './ChallengeOktaVerifyFastPassView';
+
+export default BaseAuthenticatorView.extend({
+  initialize () {
+    BaseAuthenticatorView.prototype.initialize.apply(this, arguments);
+
+    const currentAuthenticator = this.options?.appState?.get('currentAuthenticator');
+    const selectedMethod = currentAuthenticator?.contextualData?.selectedMethod;
+    if (selectedMethod === 'totp') {
+      this.Body = ChallengeOktaVerifyTotpView;
+      this.Footer = AuthenticatorVerifyFooter;
+    } else {
+      this.Body = ChallengeOktaVerifyFastPassView;
```

Comment:
@jhe-okta Not adding the footer with back link for uv device challenge to keep it consistent with m1. Please confirm that that is ok

Reply:
Yes, it is okay.